### DISCUSSION TITLE
TFT35 E3 V3.0.1 Firmware

### DISCUSSION THREAD

**Discussion by Footr** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2729] [General]
What firmware should I use on the TFT35 E3 V3.0.1 hardware?

**Comment by ghost** [comment]
i ask btt support and my chip on my e3 v3.0.1 says GD in the name so they said to use the GD version of the  TFT 35 v3 E3 if you don't have the GD version use the non GD version
